Source/driver/timestep.cpp

```diff
@@ -224,16 +216,15 @@ Castro::estdt_mhd (int is_new)
       dt3 = dt1;
 #endif
 
-      return {amrex::min(dt1, dt2, dt3)};
+      return {ValLocPair<Real, IntVect>{amrex::min(dt1, dt2, dt3), idx}};
 
-    });
+  });
 
+  if (ParallelDescriptor::IOProcessor()) {
```

Comment:
This could be misleading, because MPI communication has not been done yet and the final constraint may come from other processes.

Reply:
ah indeed... I should move that up to the caller.  But that won't explain the diffs when I run on GPUs.

Reply:
actually, I don't understand this.

Doesn't the ParReduce already do the reduction over all of the processors?  So isn't this the final value at this point?